subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junitplatform/JUnitPlatformIntegrationTest.groovy

```diff
@@ -39,6 +41,30 @@ class JUnitPlatformIntegrationTest extends JUnitPlatformIntegrationSpec {
             '''
     }
 
+    void createDynamicJupiterTest() {
```

Comment:
You are testing logging, right? If so, there is more appropriate place for such tests IMO - JUnitPlatformLoggingIntegrationTest. Consider to move your tests there